lib/rubocop/cop/rspec/empty_hook.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      #  Checks for empty before and after hooks.
+      #
+      # @example
+      #   # bad
+      #   before {}
+      #   after do; end
+      #   before(:all) do
+      #   end
+      #   after(:all) { }
+      #
+      #   # good
+      #   before { create_users }
+      #   after do
+      #     cleanup_users
+      #   end
+      #   before(:all) do
+      #     create_feed
+      #   end
+      #   after(:all) { cleanup_feed }
+      class EmptyHook < Cop
+        MSG = 'Empty hook detected.'
+
+        def_node_matcher :empty_hook?, <<~PATTERN
+          (block
+            $(send nil? #{Hooks::ALL.node_pattern_union}
+              (sym #{Hooks::Scopes::ALL.node_pattern_union})?
```

Comment:
I guess we need to extend the `Language` module with such definitions.

Reply:
But do we need to check the argument at all? Wouldn't `Hooks::ALL.block_pattern` actually do the job?

Reply:
Indeed we don't need to check the argument. `block_pattern` does not check for inner contents of the block so I am now using `Hooks::ALL.send_pattern` & empty block check with `nil?`